DMFT/src/main/java/com/gov/rsi/DMFT/controllers/AbstractController.java

```diff
@@ -0,0 +1,111 @@
+package com.gov.rsi.DMFT.controllers;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+/**
+ * Base class for all controller classes ("Controller.*"), providing common methods
+ * for response creation
+ */
+public abstract class AbstractController {
+
+    /**
+     * Generates an error response (400 Bad Request) 
+     * @return  the ResponseEntity
+     */
+    protected ResponseEntity responseBadRequest(){
+    	return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
```

Comment:
Will these ResponseEntity object be serialized into JSON for the response?

Reply:
Yes, JSON is the default serialization